mmv1/third_party/terraform/services/bigtable/resource_bigtable_table.go

```diff
@@ -270,9 +270,11 @@ func resourceBigtableTableCreate(d *schema.ResourceData, meta interface{}) error
 			if err != nil {
 				return fmt.Errorf("Error parsing automated backup policy frequency: %s", err)
 			}
-			tblConf.AutomatedBackupConfig = &bigtable.TableAutomatedBackupPolicy{
-				RetentionPeriod: abpRetentionPeriod,
-				Frequency:       abpFrequency,
+			if abpFrequency != 0 && abpRetentionPeriod != 0 { // if fields are zero this indicates disable-on-create
```

Comment:
Should `abpFrequency` be allowed to be zero here? The [docs](https://cloud.google.com/bigtable/docs/reference/admin/rest/v2/projects.instances.tables#AutomatedBackupPolicy) say "An undefined frequency is treated as 24 hours" which suggests it's allowed. That is, what is the current behavior if you try to create a table with `frequency = 0` and `retention_period = 72h`?

Reply:
Yeah, we need to know the functionality here